pkg/kubelet/dockertools/manager.go

```diff
@@ -2058,13 +2063,25 @@ func getPidMode(pod *api.Pod) string {
 func getIPCMode(pod *api.Pod) string {
 	ipcMode := ""
 	if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.HostIPC {
-		ipcMode = "host"
+		ipcMode = namespaceModeHost
 	}
 	return ipcMode
 }
 
-// GetNetNs returns the network namespace path for the given container
-func (dm *DockerManager) GetNetNs(containerID kubecontainer.ContainerID) (string, error) {
+// IsUsingHostNetwork returns true if the provided container ID is using host
+// networking.  This method is used by network plugins to determine if a pod
+// requires additional setup. Pods running with host networking do not.
+func (dm *DockerManager) IsUsingHostNetwork(containerID kubecontainer.ContainerID) (bool, error) {
```

Comment:
The reason I bring it up is that we are trying hard to isolate the runtime behind an API and the fact that the runtime details are exposed here makes it hard to support other runtimes in the future, like hyper.

@bprashanth: If you are saying that this is just an interim state, thats OK I guess (@dchen1107 WDYT?).
If this is *THE* API for now, I disagree.